app/controllers/editions_controller.rb

```diff
@@ -22,6 +22,10 @@ def show
       infoWindow: render_to_string(partial: "editions/show-editions/info_window", locals: { edition: @edition }),
       image_url: helpers.asset_url('location.png')
     }]
+
+    @edition[:avg_rating] = edition_avg_rating(@edition)
```

Comment:
Avoid using square brackets to assign attributes to an active record instance. Just use `@edition.avg_rating` instead